### PR TITLE
cli: initial commit

### PR DESCRIPTION
**Pull request by jmartinm**
* Adds reindex command that allows sending to ElasticSearch all workflows
  present in the database (based on the data type).

Signed-off-by: Javier Martin Montull <javier.martin.montull@cern.ch>

### CODE REVIEW

File: invenio_workflows_ui/cli.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+def abort_if_false(ctx, param, value):
+    """Abort command is value is False."""
+    if not value:
+        ctx.abort()
+
+
+@holdingpen.command()
+@click.option('--yes-i-know', is_flag=True, callback=abort_if_false,
+              expose_value=False,
+              prompt='Do you really want to reindex all records?')
```
Comment (david-caro) [inline invenio_workflows_ui/cli.py]: This pattern is strange for me :)

Is it different in any way to:
```python
if not yes_i_know:
    click.confirm('Do you want to continue?', abort=True)
...
```

Comment (jmartinm) [inline invenio_workflows_ui/cli.py]: Not sure, I took the pattern from https://github.com/inveniosoftware/invenio-indexer/blob/master/invenio_indexer/cli.py#L67-L89 hoping that whoever wrote the code in the first place did it for a good reason, but who knows :P

Comment (kaplun) [inline invenio_workflows_ui/cli.py]: We should not write this comments publicly :see_no_evil:

Comment (david-caro) [inline invenio_workflows_ui/cli.py]: I'd prefer to put there the simpler option though, is way more understandable. Just have a flag option that defaults to `False`.

Comment (jmartinm) [inline invenio_workflows_ui/cli.py]: Yep, just tested and it seems to be the same thing. I updated the PR

**Pull request merged**